src/lease/revoke.rs

```diff
@@ -1,44 +1,55 @@
-use crate::proto::etcdserverpb;
+use crate::protos::rpc::{LeaseRevokeRequest, LeaseRevokeResponse};
 use crate::ResponseHeader;
+use utilities::Cast;
 
 /// Request for revoking lease.
-pub struct LeaseRevokeRequest {
-    proto: etcdserverpb::LeaseRevokeRequest,
+pub struct EtcdLeaseRevokeRequest {
+    /// proto
+    proto: LeaseRevokeRequest,
 }
 
-impl LeaseRevokeRequest {
-    /// Creates a new LeaseRevokeRequest which will revoke the specified lease.
+impl EtcdLeaseRevokeRequest {
+    /// Creates a new `LeaseRevokeRequest` which will revoke the specified lease.
+    #[inline]
+    #[must_use]
     pub fn new(id: u64) -> Self {
-        let proto = etcdserverpb::LeaseRevokeRequest { id: id as i64 };
+        let proto = LeaseRevokeRequest {
+            ID: id.cast(),
+            ..LeaseRevokeRequest::default()
```

Comment:
add comma to the end of the line

Reply:
Same with previous comment, closed this one

Reply:
OK